tests/test_state/test_matcha_state.py

```diff
@@ -121,27 +123,86 @@ def experiment_tracker_state_component() -> MatchaStateComponent:
     )
 
 
-def test_state_file_getter(
-    matcha_state_service: MatchaStateService, expected_outputs: dict
+def assert_object(obj: Any, expected_type: Any) -> bool:
+    """Utility function for asserting that an object isn't None and of the correct type.
+
+    Args:
+        obj (Any): the object under test
+        expected_type (Any): the expected type of the object
+
+    Returns:
+        bool: true if the object is not None and of the expected type, otherwise false.
+    """
+    return obj and isinstance(obj, expected_type)
+
+
+def test_state_service_initialisation(
+    mock_state_file, state_file_as_object: MatchaState
```

Comment:
```
mock_state_file: None, state_file_as_object: MatchaState
```

Reply:
Is this missing from every test for a reason?

Reply:
If so, it may be a good idea for `mock_state_file` to be updated with a return value such as the location of the file as a string?

Reply:
I've set it to return a `Path` object rather than a string, to be more explicit in the typing